src/libraries/Microsoft.Extensions.Logging.Console/src/LoggerColorBehavior.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.Logging.Console
+{
+    /// <summary>
+    /// Determines when color can be enabled log messages
+    /// </summary>
+    public enum LoggerColorBehavior
+    {
+        /// <summary>
+        /// Disabled when output is redirected
```

Comment:
I think the summary should be something like `Use the default color behavior.` and then in the `<remarks>` section we can describe what the behavior is. "Enables color except when the console output is redirected."